### PR TITLE
[include] More Subjects

### PR DESCRIPTION
**Pull request by vivekmudgal**

### CODE REVIEW

File: include/subjects.json [hunk 1]
```diff
@@ -859,4 +859,11 @@
         "SCI AND TECH. RUBBER & RUBBER LIKE",
         false
     ]
```
Comment (hargup) [inline include/subjects.json]: There should be a comma here.

File: include/subjects.json [hunk 2]
```diff
@@ -859,4 +859,11 @@
...
+    "MA20101": [
+        "TRANSFORM CALCULUS"
```
Comment (hargup) [inline include/subjects.json]: Also add a comma and line with `false`. `false` indicates if we have the paper or not. Do same thing for the next subject code.

**Review by hargup — CHANGES_REQUESTED** [review]

**Comment by vivekmudgal** [comment]
@hargup Required changes made. Please review.

**Comment by athityakumar** [comment]
@hargup : LGTM now. 

@vivekmudgal001 : Merging this. Thanks for the PR! :smile:

**Pull request merged**